server/pages/index.js

```diff
@@ -388,15 +405,23 @@ function setUpLoggedInRoute( req, res, next ) {
 					}
 
 					console.error( 'API Error: ' + errorMessage );
-
-					next( error );
 				}
 
-				return;
+				return error;
 			} );
-	} else {
-		next();
+
```

Comment:
([Same as for `langPromise`](https://github.com/Automattic/wp-calypso/pull/26975/files#r217535217) here for `userPromise`.)